multi-node/aws/pkg/cluster/config.go

```diff
@@ -66,8 +120,11 @@ func DecodeConfigFromFile(out *Config, loc string) error {
 
 func NewDefaultConfig(ver string) *Config {
 	return &Config{
-		ClusterName: "kubernetes",
-		ArtifactURL: DefaultArtifactURL(ver),
+		ClusterName:  "kubernetes",
+		ArtifactURL:  DefaultArtifactURL(ver),
+		VpcCIDR:      DefaultVpcCIDR,
+		SubnetCIDR:   DefaultSubnetCIDR,
+		ControllerIP: DefaultControllerIP,
```

Comment:
In any case, you should update `cluster.go` and move the parameters into the initial definition. The if checks are only required if they can be empty.